test/User-test.js

```diff
@@ -1,12 +1,17 @@
 import { expect } from "chai";
 import User from "../src/User";
 const userTestData = require("../src/data/userTestData");
-const sleepTestData  = require("../src/data/sleepTestData");
+const sleepTestData = require("../src/data/sleepTestData");
+import Hydration from "../src/data/hydrationTestData";
```

Comment:
change the "import" to "require" as shown above for sleepTestData. import is for importing Classes and we don't have a Hydration class